features/in_onderzoek.feature

```diff
@@ -445,215 +531,246 @@ Functionaliteit: in onderzoek
       Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                             |
-      | type                | RaadpleegMetBurgerservicenummer    |
-      | burgerservicenummer | 555550001                          |
-      | fields              | burgerservicenummer,verblijfplaats |
-      Dan heeft het kind met burgerservicenummer '555550002' de volgende 'verblijfplaats.inOnderzoek' gegevens
-      | naam                             | waarde       |
-      | adresregel1                      | true         |
-      | adresregel2                      | true         |
-      | adresregel3                      | true         |
-      | land                             | true         |
-      | datumIngangGeldigheid            | true         |
-      | datumIngangOnderzoek.datum       | 2022-03-07   |
-      | datumIngangOnderzoek.type        | Datum        |
-      | datumIngangOnderzoek.langFormaat | 7 maart 2022 |
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | verblijfplaats                  |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde                   |
+      | type                                                       | VerblijfplaatsBuitenland |
+      | inOnderzoek.datumVan                                       | true                     |
+      | inOnderzoek.datumIngangGeldigheid                          | true                     |
+      | verblijfadres.regel1                                       | Rue du pomme 25          |
+      | verblijfadres.regel2                                       | Bruxelles                |
+      | verblijfadres.land.code                                    | 5010                     |
+      | verblijfadres.inOnderzoek.regel1                           | true                     |
+      | verblijfadres.inOnderzoek.regel2                           | true                     |
+      | verblijfadres.inOnderzoek.regel3                           | true                     |
+      | verblijfadres.inOnderzoek.land                             | true                     |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07               |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.type        | Datum                    |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022             |
+      | inOnderzoek.datumIngangOnderzoek.datum                     | 2022-03-07               |
+      | inOnderzoek.datumIngangOnderzoek.type                      | Datum                    |
+      | inOnderzoek.datumIngangOnderzoek.langFormaat               | 7 maart 2022             |
 
```

Comment:
Ik mis hier nog het scenario 'hele categorie verblijfplaats van een onbekend verblijfplaats is in onderzoek'

Reply:
was er initieel ook niet. En in deze p.r. heb ik bestaande 'in onderzoek' scenario's aangepast zodat zij kunnen worden gebruikt als geautomatiseerde tests